planner/cascades/testdata/transformation_rules_suite_out.json

```diff
@@ -1549,6 +1549,32 @@
       }
     ]
   },
+  {
+    "Name": "TestMergeAdjacentWindow",
+    "Cases": [
+      {
+        "SQL": "select sum(a) over(partition by b order by c), max(b) over(partition by b order by c) from t",
+        "Result": [
+          "Group#0 Schema:[Column#17,Column#18]",
+          "    Projection_8 input:[Group#1], Column#16, Column#15",
+          "Group#1 Schema:[Column#16,Column#15]",
+          "    Projection_7 input:[Group#2], Column#16, Column#15",
+          "Group#2 Schema:[test.t.a,test.t.b,test.t.c,Column#15,Column#16]",
+          "    Window_6 input:[Group#3]",
+          "Group#3 Schema:[test.t.a,test.t.b,test.t.c,Column#15]",
```

Comment:
It seems we need to add the rule `EliminateProjection` into this test case, otherwise the two Windows won't be adjacent.